### PR TITLE
UI of Account Settings Page Feature

### PR DESCRIPTION
**Pull request by fatimarafiqui**
This includes the following:
- UI template for Account Settings page
- Route for the settings page
- Minor tweaks for interface rendering

settings

### CODE REVIEW

**Review by Anjali05 — APPROVED** [review]

**Review by Anjali05 — APPROVED** [review]

**Review by Anjali05 — APPROVED** [review]

**Review by Anjali05 — APPROVED** [review]

**Review by Anjali05 — APPROVED** [review]

File: template/partial/settings.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<form class="ui form signup-form">
+	<div class="two fields">
+		<div class="field">
+			<label>Email</label>
+			<input type="text" name="email" id="email" placeholder="Email">
```
Comment (MehaKaushik) [inline template/partial/settings.html:5]: You should not keep the value of name and id as same. Probably use, emailId for id, similarly nameId in line number 10.

File: modules/Router.php [hunk 2]
```diff
@@ -230,11 +236,28 @@ public static function getPageUrl($page, $content)
     }
 
     /**
+     * Method : getSettings
+     * Description : Returns array of template and data for settings page
```
Comment (MehaKaushik) [inline modules/Router.php:240]: Please make it more descriptive, what "template" is referred here, please explain.

File: template/partial/settings.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			<div class="ui basic red right pointing prompt label transition hide"></div>
+		</div>
+		<div class="field">
+			<label>Name</label>
+			<input type="text" name="name" id="name" placeholder="Your complete name">
+			<div class="ui basic red left pointing prompt label transition hide"></div>
+		</div>
+	</div>
+	<div class="two fields">
```
Comment (MehaKaushik) [inline template/partial/settings.html:14]: Is "two fields" a good name for div? Every element name should tell what the element's usage is?

**Review by MehaKaushik — APPROVED** [review]
Please address the minor comments, before merging.

**Review by MehaKaushik — DISMISSED** [review]
Travis CI build is failing. Please check it.

File: template/partial/settings.html (hunk 1 above)
Comment (fatimarafiqui) [reply template/partial/settings.html:5]: Name and ID have different usage. Name is used by JS while ID is a style selector. Both can have same values as the selectors are different for them.
- [Name Selector](https://api.jquery.com/attribute-contains-selector/): [name*=”value”]
- [ID Selector](https://api.jquery.com/id-selector/): $(#id)

Reference: [Name vs ID](https://stackoverflow.com/questions/7470268/html-input-name-vs-id)

File: template/partial/settings.html (hunk 3 above)
Comment (fatimarafiqui) [reply template/partial/settings.html:14]: @MehaKaushik "two fields" is not a name for this div. It's a class name which is used for style sheets. These class names must be short and simple to increase page load and readability.

Also, these class names are specified in Semantic's stylesheets. We cannot really do anything with such names. Such class names are all over our codebase.

Reference: [Semantic Columns](https://semantic-ui.com/elements/container.html#/examples) | [Semantic Fields](https://semantic-ui.com/collections/form.html)

File: modules/Router.php (hunk 2 above)
Comment (fatimarafiqui) [reply modules/Router.php:240]: @MehaKaushik settings page template is referred here. It's self-explanatory that template is for the settings page. This comment is according to the other comments approved in the same file.

**Pull request merged**